src/place/mod.rs

```diff
@@ -367,3 +368,39 @@ where
             serde_json::from_value::<Vec<(f64, f64)>>(inner_arr).map_err(|e| D::Error::custom(e))
         })
 }
+
+fn serialize_bounding_box<S>(src: &Vec<(f64, f64)>, ser: S) -> Result<S::Ok, S::Error>
```

Comment:
nit: should this combine `(de)serialize_bounding_box` into a single serialize module so you can combine the serde attributes into a single `#[serde(with)]`? probably, yeah